From 64b797ac71aa582ae79d21ba25809ecb892dc747 Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Wed, 6 Mar 2024 18:00:11 +0100
Subject: [PATCH 0717/1695] dts: stm32: rename IOM EVAL pins muxing
 configurations for stm32mp25

Rename IOM EVAL pins muxing configurations to distinguish
the OSPI instances and the OSPI ports.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c165cf7ebe33bfd2a2feb352bfd904baa23067e3 dts: stm32: rename IOM EVAL pins muxing configurations for stm32mp25]
Change-Id: I58d649dd424a234d57e848d376dcc9722bb565ea
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/366665
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp25-pinctrl.dtsi | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp25-pinctrl.dtsi b/core/arch/arm/dts/stm32mp25-pinctrl.dtsi
index 97ad90cfa..6eb22fc30 100644
--- a/core/arch/arm/dts/stm32mp25-pinctrl.dtsi
+++ b/core/arch/arm/dts/stm32mp25-pinctrl.dtsi
@@ -23,7 +23,7 @@
 		};
 	};
 
-	ospi1_clk_pins_a: ospi1-clk-0 {
+	ospi_port1_clk_pins_a: ospi-port1-clk-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 0, AF10)>; /* OSPI1_CLK */
 			bias-disable;
@@ -32,13 +32,13 @@
 		};
 	};
 
-	ospi1_clk_sleep_pins_a: ospi1-clk-sleep-0 {
+	ospi_port1_clk_sleep_pins_a: ospi-port1-clk-sleep-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 0, ANALOG)>; /* OSPI1_CLK */
 		};
 	};
 
-	ospi1_cs0_pins_a: ospi1-cs0-0 {
+	ospi_port1_cs0_pins_a: ospi-port1-cs0-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 3, AF10)>; /* OSPI_NCS0 */
 			bias-pull-up;
@@ -47,13 +47,13 @@
 		};
 	};
 
-	ospi1_cs0_sleep_pins_a: ospi1-cs0-sleep-0 {
+	ospi_port1_cs0_sleep_pins_a: ospi-port1-cs0-sleep-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 3, ANALOG)>; /* OSPI_NCS0 */
 		};
 	};
 
-	ospi1_io03_pins_a: ospi1-io03-0 {
+	ospi_port1_io03_pins_a: ospi-port1-io03-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 4, AF10)>, /* OSPI_IO0 */
 				 <STM32_PINMUX('D', 5, AF10)>, /* OSPI_IO1 */
@@ -65,7 +65,7 @@
 		};
 	};
 
-	ospi1_io03_sleep_pins_a: ospi1-io03-sleep-0 {
+	ospi_port1_io03_sleep_pins_a: ospi-port1-io03-sleep-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 4, ANALOG)>, /* OSPI_IO0 */
 				 <STM32_PINMUX('D', 5, ANALOG)>, /* OSPI_IO1 */
-- 
2.47.2

